modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -11897,12 +11897,8 @@
                                   "phoneNumber": {
                                     "description": "Phone number.",
                                     "type": "string",
-                                    "pattern": "^[0-9]{1,14}$",
+                                    "pattern": "^\\d(?:[- ]*\\d){1,14}$",
```

Comment:
This regex feels much more permissive than we likely want. It requires one digit to start and then from one to fourteen repeats of digit, dash-digit, or space-digit.

What is valid that I assume shouldn't be:
* `12`
* `1-2-3`
* `1-2-3-4-5 6 789 0-1-2-3 45`

We don't have to solve all of these with an adjustment to the regex, we can also use `minLength` and `maxLength` string properties to enforce length limits.